tooltester/src/main/resources/PipelineTest.groovy

```diff
@@ -22,23 +22,34 @@ def transformIntoStep(url, tag, descriptor, parameter, entryType, synapseCache)
                 sh 'rm -rf /mnt/output/*'
                 sh 'rm -rf /media/large_volume/output/*'
                 step([$class: 'WsCleanup'])
+                sh "wget https://raw.githubusercontent.com/ga4gh/dockstore-support/feature/setRunners/tooltester/src/main/resources/${AnsiblePlaybook}.yml"
```

Comment:
Watch out, this looks like a branch reference which will disappear after a merge